src/webserver/src/WebServer.cpp

```diff
@@ -530,6 +530,8 @@ void CWebServerBase::Send_Server_Cmd(uint32 ip, uint16 port, wxString cmd)
 	if ( ec_cmd && ip ) {
 		ec_cmd->AddTag(CECTag(EC_TAG_SERVER, EC_IPv4_t(ip, port)));
 		Send_Discard_V2_Request(ec_cmd);
+	}
+	if ( ec_cmd ) {
```

Comment:
This `if` is useless, as [deleting a null pointer is a no-op](https://en.cppreference.com/w/cpp/language/delete).